src/cpp/src/visual_language/continuous_batching_adapter.hpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "visual_language/pipeline_base.hpp"
+#include "openvino/genai/continuous_batching_pipeline.hpp"
+
+using namespace ov::genai;
+
+class ov::genai::VLMPipeline::VLMContinuousBatchingAdapter : public ov::genai::VLMPipeline::VLMPipelineBase {
+public:
+    ContinuousBatchingPipeline m_impl;
+
+    VLMContinuousBatchingAdapter(
+        const std::filesystem::path& models_dir,
+        const SchedulerConfig& scheduler_config,
+        const std::string& device,
+        const ov::AnyMap& properties
+    ): m_impl{
+        models_dir, 
+        scheduler_config, 
+        device, 
+        properties} { }
+
+    VLMContinuousBatchingAdapter(
+        const ModelsMap& models_map,
+        const Tokenizer& tokenizer,
+        const std::filesystem::path& config_dir_path,
+        const SchedulerConfig& scheduler_config,
+        const std::string& device,
+        const ov::AnyMap& properties,
+        const ov::genai::GenerationConfig& generation_config
+    ): m_impl{
+        "./", 
```

Comment:
looks like in future we have to replicate the same ctor in CB pipeline..

Let's keep as TODO right now.

Reply:
Added TODO.

Reply:
PR https://github.com/openvinotoolkit/openvino.genai/pull/1863